en/_posts/2024-06-10-crisis-toolkit.md

```diff
@@ -0,0 +1,98 @@
+<li>Conditions and policies change frequently, requiring frequent updates while keeping content digestible. Presenting relevant content in plain language and all in one place is key.</li>
+</ul>
+
+<p>The following resources provide tips on how to make content concise, clear, and accessible.</p>
+
+<ul>
+<li><a href="https://design.canada.ca/crisis/content.html">Crisis communications content design checklist</a></li>
+<li><a href="https://blog.canada.ca/2020/10/21/using-graphics.html">Using graphics to bridge language gaps</a></li>
+<li><a href="https://blog.canada.ca/2020/06/05/designing-for-accessibility.html">Designing for accessibility during COVID-19</a></li>
+<li><a href="https://blog.canada.ca/2020/07/15/alert-fatigue">Alert fatigue during COVID-19</a></li>
+</ul>
+
+<h2>Provide answers, not information</h2>
+<ul>
+<li>In a crisis situation, people are overwhelmed and face an avalanche of new information.</li>
+<li>When seeking answers to questions on Canada.ca, they need to feel confident that they’ve found a complete answer that is relevant to their task or situation.</li>
+<li>Interactive content simplifies the task of finding answers and guides users to what they need to know in order to complete a task, without needing to “forage” for information across multiple pages.</li>
+<li>Be ready to coordinate and provide links to provincial resources. Here’s an example of how that was done for COVID vaccinations:
+<ul>
+ <li><a href="https://www.canada.ca/en/public-health/services/diseases/coronavirus-disease-covid-19/vaccines/how-vaccinated.html#a1">How to get vaccinated: Book a vaccination appointment</a></li>
+ </ul></li>
+</ul>
+<p>To learn more about how to use answers instead of information, read:</p>
+<ul>
+<li><a href="https://design.canada.ca/research-summaries/travel-restrictions-study.html">Using a wizard to help people get answers during COVID-19</a></li>
+<li><a href="https://blog.canada.ca/2020/11/12/interactive-checklists.html">Using interactive checklists to simplify eligibility criteria</a></li>
+<li><a href="https://blog.canada.ca/2021/04/08/using-interactive-questions.html">The magic of using interactive questions</a></li>
+</ul>
+
+<h2>Creating a structure so people can find content</h2>
+<ul>
+<li>Just because information is available doesn’t mean it will be found or read.</li>
+<li>Using familiar terms in links and making navigation pages easy to scan helps users find what they’re looking for more quickly and accurately.</li>
+<li>In a crisis, people are more likely to visit from their phones. Design the navigation for mobile, rather than favouring desktop.</li>
+</ul>
+
+<p>For more information about creating structure to help people find content, read:</p>
+<ul>
+<li>Flexible navigation page <a href="https://docs.google.com/document/d/1AXU_Igv6ZAkVOifpMgwld0zQNXSzib0aSxpSIdsI0lw/edit#heading=h.y8x0q33ditvt">(draft)</a></li>
```

Comment:
Please remove: (draft)

Please add the link to the Flexible navigation blog post, it is: https://blog.canada.ca/2024/05/13/flexible-navigation-page-design-for-emergencies.html